adhocracy4/projects/models.py

```diff
@@ -145,6 +145,18 @@ def other_projects(self):
     def is_private(self):
         return not self.is_public
 
+    @property
+    def modules(self):
+        return self.module_set
+
+    @functional.cached_property
+    def active_module(self):
```

Comment:
Should we name that differently, because of the difference between an active module and an active phase in the proposed implementation? Like `last_active_module'?